includes/validation/class-amp-validation-error-taxonomy.php

```diff
@@ -2141,26 +2147,43 @@ public static function render_single_url_error_details( $validation_error, $term
 				if ( $is_element_attributes && empty( $value ) ) {
 					continue;
 				}
-				if ( in_array( $key, [ 'code', 'type', 'property_value', 'mandatory_anyof_attrs', 'mandatory_oneof_attrs' ], true ) ) {
+				if ( in_array( $key, [ 'code', 'type', 'css_property_value', 'mandatory_anyof_attrs', 'meta_property_value', 'meta_property_required_value', 'mandatory_oneof_attrs' ], true ) ) {
 					continue; // Handled above.
 				}
+				if ( 'spec_name' === $key ) {
+					continue;
+				}
 				?>
 				<dt><?php echo esc_html( self::get_source_key_label( $key, $validation_error ) ); ?></dt>
 				<dd class="detailed">
 					<?php if ( in_array( $key, [ 'node_name', 'parent_name' ], true ) ) : ?>
 						<code><?php echo esc_html( $value ); ?></code>
-					<?php elseif ( 'property_name' === $key ) : ?>
+					<?php elseif ( 'css_property_name' === $key ) : ?>
 						<?php
-						if ( isset( $validation_error['property_value'] ) && is_scalar( $validation_error['property_value'] ) ) {
+						if ( isset( $validation_error['css_property_value'] ) && is_scalar( $validation_error['css_property_value'] ) ) {
 							printf(
 								'<code>%s: %s</code>',
 								esc_html( $value ),
-								esc_html( $validation_error['property_value'] )
+								esc_html( $validation_error['css_property_value'] )
 							);
 						} else {
 							printf( '<code>%s</code>', esc_html( $value ) );
 						}
 						?>
+					<?php elseif ( 'meta_property_name' === $key ) : ?>
```

Comment:
As we're using these array keys in multiple places, I would suggest using constants for them.

However, given that the long-term plan is to use objects for validation errors, this might just be a waste of effort at this point.